content/guides/applications/web-app-integrations/configure.md

```diff
@@ -38,42 +38,32 @@ To configure the integration, follow the guidance below for each value.
 
 <!-- markdownlint-disable line-length -->
 
-| Field                     | Description                                                                                                                                                                                                                      |
+| Field  Description |
 |---------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Integration name          | The name of your integration, which users see in the Box Web App when the ** More Options** > **Integrations** menu is selected on a file or folder.                                                                             |
-| Description               | The description of the integration displayed in the Box App Center.                                                                                                                                                             |
-| Supported file extensions | The integration will only appear as an option in the ** More Options** > **Integrations** menu for the selected file extensions.                                                                                                 |
-| Display on shared pages   | Determines if an integration can be shown to external users on a shared page. If enabled, users who are not collaborating on the content will see the integration in the context-menu when browsing items through a shared link. |
+| Integration name | The name of your integration, which users see in the Box Web App when they select the **More Options** > **Integrations** menu on a file or folder. |
+| Description | The description of the integration displayed in the Box App Center. |
+| Supported file extensions | The integration will only appear as an option in the **More Options** > **Integrations** menu for the selected file extensions. |
+| Permissions requirement | Determines what permissions level users need to see the integration. Option **Download permissions are required** allows users to download the file - they will not be able to update it. Option **Full permissions are required** allows users to download and update the file. |
+| Integration Scopes | Specifies the scope of your integration. |
+| Display on shared pages toggle| Determines if an integration can be shown to external users on a shared page. If enabled, users who are not collaborating on the content will see the integration in the context-menu when browsing items through a shared link. |
+| Lock to only allow the current user to overwrite the file using your integration toggle | Determines if different web app integrations can edit the file at the same time. |
+| Integration type | Select the correct integration type. Available options are: **Files**, **Folders**, **Both**. |
```

Comment:
I think each of them is correct - so maybe we can skip this word or replace it with something similar.